src/main/java/org/phyloref/jphyloref/commands/TestCommand.java

```diff
@@ -42,77 +49,90 @@
  *
  */
 public class TestCommand implements Command {
     /**
-     * This command is named Test. It should be 
+     * This command is named "test". It should be
      * involved "java -jar jphyloref.jar test ..."
      */
 
     @Override
-    public String getName() { 
-        return "test"; 
+    public String getName() {
+        return "test";
     }
-    
+
     /**
      * A description of the Test command.
-     * 
+     *
      * @return A description of this command.
      */
     @Override
-    public String getDescription() { 
-        return "Test the phyloreferences in the provided ontology to determine if they resolved correctly."; 
+    public String getDescription() {
+        return "Test the phyloreferences in the provided ontology to determine if they resolved correctly.";
     }
 
     /**
-     * Add command-line options specific to this command. There is only one:
-     * -i or --input can be used to set the RDF/XML file to read.
-     * 
+     * Add command-line options specific to this command.
+     *
      * @param opts The command-line options to modify for this command.
      */
     @Override
     public void addCommandLineOptions(Options opts) {
         opts.addOption(
-            "i", "input", true, 
+            "i", "input", true,
             "The input ontology to read in RDF/XML (can also be provided without the '-i')"
         );
+        opts.addOption(
+            "nr", "no-reasoner", false,
+            "Turn off reasoning (all tests will fail!)"
+        );
     }
 
     /**
      * Given an input ontology, reason over it and determine if nodes are
-     * identified correctly. It provides output using the Test Anything 
-     * Protocol (TAP: https://testanything.org/). 
-     * 
+     * identified correctly. It provides output on System.out using the
+     * Test Anything Protocol (TAP: https://testanything.org/).
+     *
      * @param cmdLine The command line options provided to this command.
      */
     @Override
     public void execute(CommandLine cmdLine) throws RuntimeException {
-        // Determine which input ontology should be read, 
+        // Extract command-line options
         String str_input = cmdLine.getOptionValue("input");
+        boolean flag_no_reasoner = cmdLine.hasOption("no-reasoner");
 
         if(str_input == null && cmdLine.getArgList().size() > 1) {
             // No 'input'? Maybe it's just provided as a left-over option?
-            str_input = cmdLine.getArgList().get(1); 
+            str_input = cmdLine.getArgList().get(1);
         }
 
         if(str_input == null) {
             throw new RuntimeException("Error: no input ontology specified (use '-i input.owl')");
         }
 
+        // Create File object to load
         File inputFile = new File(str_input);
         if(!inputFile.exists() || !inputFile.canRead()) {
             throw new RuntimeException("Error: cannot read from input ontology '" + str_input + "'");
```

Comment:
Why test this? IO operations on this file _will_ fail, and with the appropriate exception, right?

Reply:
I'm used to APIs failing awkwardly if you give them wrong input, but you're right -- the OWLAPI throws a nice FileNotFoundException if the file doesn't exist. I'll get rid of this code!